TeamShiksha/logoexecutive
# Reduce env file to less than 4KB.

### Summary
closes #327 
Reduced the env variables and removed serviceAccountKey.json


### Description
- Moved some of the environment variables to constants and parameterized it using existing envs.
- Removed the logic of reading envs from serviceAccountKey.json.


### How to Test the Changes
Run the branch locally and check all the apis are working properly and try to create text file where you assign all the proper values to env mentioned in env_example and check the size of the file.


### Context (Optional)

### Screenshots or Recordings (Optional)


![envsize](https://github.com/TeamShiksha/logoexecutive/assets/88454618/fafef00a-f296-4f47-9bf6-0414ccd155b9)

tested few main apis and they worked fine

https://github.com/TeamShiksha/logoexecutive/assets/88454618/388276b4-343e-4da9-a6d2-e41c4e734cf2



![part-1](https://github.com/TeamShiksha/logoexecutive/assets/88454618/99eaaccf-9ea2-4464-9c6a-9d799482360f)
![part-2](https://github.com/TeamShiksha/logoexecutive/assets/88454618/d75ec20d-ede5-4e2d-aa15-abe8ebbc52bc)



## Checklist
